### PR TITLE
added single video page

### PR DESCRIPTION
**Pull request by JantuDeb**
# Single Video page.
- Play video, pause, resume.
- recommended videos in the right sidebar (for now showing a few videos from the list)
- view count, like count, publish date (not functional for now)

image

### CODE REVIEW

File: src/utils/utils.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export const formatedDate = (date) =>
+new Intl.DateTimeFormat("en-IN", { dateStyle: "medium" }).format(
+  new Date(date)
+);
```
Comment (tushartiwari7) [inline src/utils/utils.js:4]: Liked How you formatted the Date here.

**Review by tushartiwari7 — APPROVED** [review]
The UI of Video page is very good.
I liked the way you used to format the Date.
You Can merge this.

File: src/pages/video-details/VideoDetails.jsx [hunk 2]
```diff
@@ -1,75 +1,77 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 import Navbar from "../../components/navbar/Navbar";
 import VideoPlayer from "../../components/player/VideoPlayer";
 import HorizontalVideoCard from "../../components/shared/HorizontalVideoCard";
```
Comment (Bhakti199) [inline src/pages/video-details/VideoDetails.jsx:4]: you may export this from single `index` file.

**Review by Bhakti199 — APPROVED** [review]
- This code looks fine to me.
- Well written and structured.
- Proper defining of functions for repeating task.
- Liked the use of  dateTimeFormat.
- Everything looks cool.
- This PR is good to me.

**Review by neetukumari4858 — APPROVED** [review]
-I liked color Theam it is in medium not very dark and not very light.
-code is well Structured.
-UI looks good.
-code is readable.

**Review by ShreyashMahajan — APPROVED** [review]
Things which I liked.
1. Your UI is absolutely amazing, clean, and clear.
2.  Code is very understandable.
3.  It is written in proper hierarchical order.

**Pull request merged**